.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,25 @@
+name: Semgrep
+on:
+  workflow_dispatch: {}
+  pull_request: {}
+  push:
+    branches:
+      - main
+      - master
+    paths:
+      - .github/workflows/semgrep.yml
```

Comment:
This will only run on push to this exact file, is that the desired behavior?

Reply:
No it's not. There should be no path set since we want to run it on everything except E2E tests but that'll be configured in the semgrep config

Reply:
We typically like to run all CI again on push to develop as the merged result may be different than the PR contents.